package.json

```diff
@@ -115,7 +115,7 @@
     ]
   },
   "resolutions": {
-    "webpack": "^5.59.0",
-    "@carbon/react": "1.26.0"
```

Comment:
IIRC this is needed to ensure that yarn doesn't downgrade the version to one compatible with `gatsby-theme-carbon`. The theme is almost always lagging behind in what version of Carbon it depends on. Maybe with yarnv3 the resolution/dedupe algo is better though and we won't run into issues, I'm not sure.